packages/api/lambdas/sqs-message-consumer.js

```diff
@@ -18,66 +18,95 @@ const Rule = require('../models/rules');
  * messages from SQS queue are processed
  */
 async function processQueues(event, dispatchFn) {
-  const model = new Rule();
-  const rules = await model.getRulesByTypeAndState('sqs', 'ENABLED');
+  const rulesModel = new Rule();
+  const rules = await rulesModel.queryRules({
+    type: 'sqs',
+    state: 'ENABLED'
+  });
 
   const messageLimit = event.messageLimit || 1;
   const timeLimit = event.timeLimit || 240;
 
-  await Promise.all(rules.map(async (rule) => {
+  const rulesByQueueMap = rules.reduce((map, rule) => {
```

Comment:
Super nitpicky, so don't change it, but I feel like this could be easier to understand as a separate function with separate tests.